python/nano/src/bigdl/nano/deps/openvino/pytorch/model.py

```diff
@@ -45,6 +45,9 @@ def __init__(self, model, input_sample=None, **export_kwargs):
             OpenVINOModel.__init__(self, ov_model_path)
             AcceleratedLightningModule.__init__(self, None)
 
+    def __call__(self, *inputs, **kwargs):
+        return AcceleratedLightningModule.__call__(self, *inputs, **kwargs)
```

Comment:
does this call go to `LightningModule`?

Reply:
`AcceleratedLightningModule.__call__` will call `__call__` method of `LightningModule`

Reply:
Does it make sense? Should it call `AcceleratedLightningModule.forward`

Reply:
`LightningModule.__call__` will call `AcceleratedLightningModule.forward` because `AcceleratedLightningModule` inherits from `LightningModule` and overrides the `forward` function.